llvm/test/CodeGen/X86/fold-int-pow2-with-fmul-or-fdiv.ll

```diff
@@ -111,6 +111,72 @@ define <4 x float> @fmul_pow2_ldexp_4xfloat(<4 x i32> %i) {
 ; CHECK-AVX-NEXT:    addq $40, %rsp
 ; CHECK-AVX-NEXT:    .cfi_def_cfa_offset 8
 ; CHECK-AVX-NEXT:    retq
+;
+; CHECK-AVX512F-LABEL: fmul_pow2_ldexp_4xfloat:
```

Comment:
Doesn't it appeared because of dropped `CHECK-AVX` prefix from `-mattr=+avx512f` RUN line? Can we return it back? I think the same is for `CHECK-SKX`.